core/api/src/app/accounts/create-account.ts

```diff
@@ -51,6 +55,11 @@ const initializeCreatedAccount = async ({
   account.statusHistory = [{ status: config.initialStatus, comment: "Initial Status" }]
   account.level = config.initialLevel
 
+  // Set display currency to match phone numbers country code
+  const countryCode = (phoneMetadata?.countryCode || "US") as Countries
+  const currency = countryToCurrency[countryCode]
+  account.displayCurrency = currency as DisplayCurrency
```

Comment:
we need to validate against our list of currencies (we dont support all the currencies)